Automattic/wp-calypso
# Stepper: Async load user on start

## Proposed changes

Currently the stepper is fetching the user and waiting until the user data is available to render the first step. Since we don't need the user info for the 1st step, this PR removes the blocking request and renders the 1st step and fetches the user data without blocking the stepper.

## How to test

Nothing should really change for the user, this change is mainly a performance fix

Pull and run this branch
Navigate to `/setup/newsletter` or `/setup/link-in-bio`
The flow should work normally if logged in and logged out

## Perf difference
https://www.webpagetest.org/video/compare.php?tests=230406_AiDc70_6FP,230406_AiDc80_6FQ

Related to https://github.com/Automattic/wp-calypso/issues/75215
